config/config.tyndp.yaml

```
@@ -40,6 +40,20 @@ co2_budget:
 electricity:
   base_network: tyndp-raw
   transmission_limit: v1.0
+  pecd_renewable_profiles:
+    enable: true
+    technologies:
+      Wind_Offshore:
+      - offwind-ac-fb-r
+      - offwind-ac-fl-r
+      - offwind-dc-fb-r
+      - offwind-dc-fl-r
+      - offwind-dc-fb-oh
+      - offwind-dc-fl-oh
+      - offwind-h2-fb-oh
+      - offwind-h2-fl-oh
+
+  renewable_carriers: [solar, solar-hsat, onwind, offwind-ac-fb-r, offwind-ac-fl-r, offwind-dc-fb-r, offwind-dc-fl-r, offwind-dc-fb-oh, offwind-dc-fl-oh, offwind-h2-fb-oh, offwind-h2-fl-oh, hydro]
```

Maintaining the same order as in the `config.default.yaml`

```suggestion
  renewable_carriers: [solar, solar-hsat, onwind, offwind-ac-fb-r, offwind-ac-fl-r, offwind-dc-fb-r, offwind-dc-fl-r, offwind-dc-fb-oh, offwind-dc-fl-oh, offwind-h2-fb-oh, offwind-h2-fl-oh, hydro]
  
  pecd_renewable_profiles:
    enable: true
    technologies:
      Wind_Offshore:
      - offwind-ac-fb-r
      - offwind-ac-fl-r
      - offwind-dc-fb-r
      - offwind-dc-fl-r
      - offwind-dc-fb-oh
      - offwind-dc-fl-oh
      - offwind-h2-fb-oh
      - offwind-h2-fl-oh
```